src/planscape/forsys/get_forsys_inputs.py

```diff
@@ -0,0 +1,123 @@
+from django.contrib.gis.geos import MultiPolygon, Polygon
+from django.http import QueryDict
+from typing import TypedDict
+
+import json
+
+
+# A list of coordinates representing a polygon.
+class PolygonFromUrlParams(TypedDict):
+    coordinates: list[tuple[float, float]]
+
+
+# A project area composed of multiple disjoint polygons.
+class ProjectAreaFromUrlParams(TypedDict):
+    # Project ID
+    id: int
+    # SRID
+    srid: int
+    # Disjoint polygons that are part of the project area.
+    polygons: list[PolygonFromUrlParams]
+
+
+# Gathers forsys input parameters from url params, database lookups, or a
+# combination of the two.
+# Of note, the option to set all forsys input paramters via url parameters is
+# intended for backend debugging purposes while the option to set most forsys
+# input parameters via database lookups is intended for production.
+class ForsysScenarioSetRequestParams():
+    # Constants for parsing url parameters.
+    _URL_USE_ONLY_URL_PARAMS = 'set_all_params_via_url_with_default_values'
+    _URL_REGION = 'region'
+    _URL_PRIORITIES = 'priorities'
+    _URL_PROJECT_AREAS = 'project_areas'
+
+    # Constants that act as default values when parsing url parameters.
+    _DEFAULT_REGION = 'sierra_cascade_inyo'
+    _DEFAULT_PRIORITIES = ['fire_dynamics',
+                           'forest_resilience', 'species_diversity']
```

Comment:
how are we picking defaults here?

Reply:
i think we also have default values for constraints

Reply:
The defaults are just manual values that seem to work when testing the main flow.

I haven't implemented constraints yet but agreed: it'll be useful to have defaults for them, too.

Reply:
Ack, we should probably remove these as we get the front end working. The defaults should come from the front end I think (which might come from somewhere else in the backend)

Reply:
I kind of agree.
It depends on how we want to sanity check the backend - at the moment, I've been sanity checking the backend without bringing up a frontend at all.
Is there a TODO to add for this?

Reply:
Ok to leave as is for the moment. I think as we get more FE development, we'll naturally fill this in. Or we could leave that as the TODO haha